UPGRADE.md

```diff
@@ -0,0 +1,60 @@
+# Upgrade Notes
```

Comment:
This file should likely be splitted in the existing documentation pages:

- the new stuff into 12.3.0 in a one entry per line format
https://github.com/pimcore/pimcore/blob/12.x/doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

- the detailed steps and things that devs must kept in mind before and when upgrading in
https://github.com/pimcore/pimcore/blob/12.x/doc/23_Installation_and_Upgrade/07_Updating_Pimcore/14_V12_to_V13.md

or there was a new idea to have a way to sort of officially share our plans in the repo root? if  so, then it would be fine